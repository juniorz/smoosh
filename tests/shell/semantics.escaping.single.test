# cf tp399
cat <<weirdo
line one
line two
"line".\${PATH}.\'three\'\\x\
line four
weirdo
