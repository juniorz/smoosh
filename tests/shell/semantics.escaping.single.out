line one
line two
"line".${PATH}.\'three\'\xline four
